src/main/scala/diplomacy/LazyModule.scala

```diff
@@ -374,12 +373,6 @@ sealed trait LazyModuleImpLike extends RawModule {
       _ ()
     }
 
-    if (wrapper.shouldBeInlined) {
-      chisel3.experimental.annotate(new ChiselAnnotation {
-        def toFirrtl = InlineAnnotation(toNamed)
-      })
-    }
-
```

Comment:
please keep this, see circt [doc](https://circt.llvm.org/docs/Passes/#-firrtl-inliner-performs-inlining-flattening-and-dead-module-elimination) here. it will still be available